.env.test

```diff
@@ -25,7 +25,7 @@ PRIVATE_KEY_PATH=./test/setup/test-key.pem
 ATLASSIAN_URL=https://test-atlassian-instance.net
 TRACKING_DISABLED=true
 HYDRO_BASE_URL=https://hydro-base-url.com/api/v1/events
-HYDRO_APP_SECRET=2dd220c366ec5b86104efd7324c2d405
+HYDRO_APP_SECRET=2dd2testsecret2d405
```

Comment:
Highlighting that it is not a real secret

Reply:
We can remove the HYDRO stuff completely!

Reply:
I tried but tests fail without it

Reply:
I think we probably want to look at ripping HYDRO out as a separate issue. Maybe create a Jira issue and follow up later?